setup.py

```diff
@@ -1,14 +1,14 @@
 from setuptools import setup
 from torch.utils.cpp_extension import BuildExtension, CUDAExtension
 import os
-from make import build
```

Comment:
The setup is used also in the CI/CD pipeline to create the precompiled packages so I would keep both the Linux version with the `make.py` file and the new Windows version ad add an `if os.name=='posix'` to choose between the two versions of the code

Reply:
Ah I see. I will add the `if os.name=='posix'`.
Since this will include setting `cuda_home = os.getenv("CUDA_HOME", "/usr/local/cuda")` again, what do you think about replacing this with `from torch.utils.cpp_extension import CUDA_HOME`? On Manjaro, for example, the path to cuda is `CUDA_PATH="/opt/cuda"` and `CUDA_HOME` is not set. The torch import would take care of that.

Reply:
Perfect, thanks.
Replacing the CUDA_HOME definition seems a good idea!